### ISSUE TITLE
[Bug] Crashes with Fabric API 0.60.0

### ISSUE THREAD

**Issue by lunarbunten** [quiqueck/BCLib#33]
### What happened?

AFAIK, some of the internal Fabric API classes referenced for BCLib API have been changed and currently cause crashes.

### BCLib

2.0.20

### Fabric API

0.60.0

### Fabric Loader

0.14.9

### Minecraft

1.19.2

### Relevant log output

```shell
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'betterend'!
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:459)
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.NoClassDefFoundError: net/fabricmc/fabric/mixin/object/builder/SpawnRestrictionAccessor
	at org.betterx.bclib.api.v2.spawning.SpawnRuleBuilder.build(SpawnRuleBuilder.java:316)
	at org.betterx.bclib.api.v2.spawning.SpawnRuleBuilder.buildNoRestrictions(SpawnRuleBuilder.java:325)
	at org.betterx.betterend.registry.EndEntities.register(EndEntities.java:89)
	at org.betterx.betterend.BetterEnd.onInitialize(BetterEnd.java:42)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
	... 8 more
Caused by: java.lang.ClassNotFoundException: net.fabricmc.fabric.mixin.object.builder.SpawnRestrictionAccessor
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:145)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	... 13 more
```

**Comment by quiqueck** [comment]
Thanks for notifying me. It is not internal FAPI, but a deprecated and now removed one. Will be fixed soon.

**Comment by WaterishGithub** [comment]
oh cool so is it fixed now, or do we have to wait for one?

**Comment by quiqueck** [comment]
Yes, the fix is uploaded to CurseForge (BCLib 2.1.0)
